### ISSUE TITLE
Error al cobrar desde marketplace

### ISSUE THREAD

**Issue by emleonardelli** [mercadopago/dx-php#207]
> For support request no related with bugs or feature requests for this SDK, please use the official developers support channel https://mercadopago.com.ar/developers/es/support/

**Describe the bug**

**To Reproduce**
2. Using this snippet '....' 
3. See error

**Expected behavior**

**Screenshots**

**Server (please complete the following information):**
 - WebServer

**Additional context**

**Comment by gusgeek** [comment]
Estuve viendo un poco el SDK, por lo que entiendo sin ser DEV de MELI, no estaria siendo compatible el SDK con este metodo.

**Comment by emleonardelli** [comment]
> Estuve viendo un poco el SDK, por lo que entiendo sin ser DEV de MELI, no estaria siendo compatible el SDK con este metodo.

Efectivamente, y trate de usar su CURL pero tampoco, la verdad que estoy muy atrasado con este issue

Ni bien obtenga una respuesta la voy a dejar en este foro

Muchisimas gracias por tu ayuda @gusgeek

**Comment by gusgeek** [comment]
proba con mi sdk https://github.com/gusgeek/SAPIDK-MercadoPago

**Comment by emleonardelli** [comment]
> proba con mi sdk https://github.com/gusgeek/SAPIDK-MercadoPago

Mil gracias por tu respuesta @gusgeek te aviso ni bien testee

**Comment by phgoncalves** [comment]
Hola, @emleonardelli 
Hice algunas pruebas y pude generar un cobro de Marketplace.
Puedes compartir las llamadas que hiciste para generar el token? Así pruebo todo el flujo.
